ui/src/containers/Layout/style.module.scss

```diff
@@ -1,59 +1,136 @@
 @import 'src/scss/lib';
 
 .container {
+  display: flex;
+  height: 100%;
+  min-height: 100vh;
+  position: relative;
+  width: 100vw;
+}
+
+.sidebar {
   align-items: center;
-  margin: 0 auto;
-  max-width: 1024px;
-  padding-top: 66px;
+  border-right: 1px solid $color-button-border;
+  display: flex;
+  flex-direction: column;
   position: relative;
+  width: 220px;
 }
 
 .header {
   align-items: center;
-  background-color: transparent;
-  border-bottom: 1px solid transparent;
   display: flex;
-  left: 0;
-  padding: 0 2em;
-  position: fixed;
-  right: 0;
-  top: 0;
-  transition: border-bottom 150ms ease 0s, top 250ms ease 0s, background-color 300ms ease 0s;
-  z-index: 2;
-
-  nav {
-    align-items: center;
-    display: flex;
-    flex-grow: 1;
-    flex-shrink: 0;
-    font-size: 14px;
-    height: 66px;
-    justify-content: space-between;
-    margin: 0 auto;
-    max-width: 1024px;
+  margin: 9px 21px 0 20px;
+  width: 179px;
+}
+
+.dot-menu-icon {
+  color: $color-text-dim2;
+  cursor: pointer;
+  height: 22px;
+  margin-left: 12px;
+  width: 22px;
+}
+
+.detail {
+  align-items: center;
+}
+
+.logo-image {
+  border-radius: 50%;
+  cursor: pointer;
+  height: 20px;
+  width: 20px;
+}
+
+.pic-container {
+  display: flex;
+  flex-direction: row-reverse;
+  height: 18px;
+  margin-left: auto;
+  width: 18px;
+
+  .pic {
+    border-radius: 50%;
   }
 
-  .brand {
-    color: $color-text-bright1;
-    font: inherit;
-    font-size: 1.4em;
-    font-weight: bold;
-    text-decoration: none;
+  .online-status {
+    align-self: flex-end;
+    background: $color-online;
+    border: 1px solid $color-bg-dark;
+    border-radius: 50%;
+    height: 8px;
+    margin-left: 1px;
+    margin-top: 1px;
+    position: absolute;
+    width: 8px;
   }
+}
+
+.admin-section {
+  color: $color-text-dim2;
+  display: grid;
+  font-size: 12px;
+  grid-auto-rows: minmax(1.5em, auto);
+  grid-row-gap: 8px;
+  grid-template-columns: 1fr;
+  line-height: 15px;
+  margin-top: 2em;
+  width: 81%;
+}
+
+.organization-container {
+  cursor: pointer;
+  grid-column: 1;
+  grid-row: 2;
+}
 
-  .logo {
-    margin: 0 4px;
-    width: 0.8em;
+.admin-section-title {
+  color: $color-text-primary;
+  font-size: 13px;
+  line-height: 16px;
+  margin-left: 10px;
+}
+
+.employee-container {
+  cursor: pointer;
+  grid-column: 1;
+  grid-row: 3;
+}
```

Comment:
Sidebar seem like an appropriate place to use flexbox since it requires a single-dimensional layout.